tests/button.test.tsx

```diff
@@ -2,10 +2,11 @@ import { default as Button } from '../src/app/components/Button';
 import { render } from '@testing-library/react';
 import '@testing-library/jest-dom';
 
-describe('Example test suite', () => {
-  it('should pass', () => {
+describe('Button component', () => {
+  it('should render', () => {
```

Comment:
Maybe let's create incorrect test naming? i.e. 

`it('expect to render')`? :)

Reply:
Sure, can change it 😃